app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -163,6 +256,57 @@ export default function Navigation() {
     </div>
   );
 
+  const menuItems = (
+    <div>
+      {menuDropDown(data).map(
+        ({ name, title, notificationCount, route, target }) => {
+          if (title)
+            return (
+              <MenuItem classes={{ root: classes.menuTitle }}>{name}</MenuItem>
+            );
+
+          const nameElement = <Typography noWrap>{name}</Typography>;
+          const hasNotification =
+            notificationCount !== undefined && notificationCount > 0;
+
+          return (
+            <MenuItem key={name}>
+              <a
+                href={route}
+                target={target || "_self"}
+                className={`${classes.menuItem} ${
+                  hasNotification && classes.menuItemMessage
+                }`}
+              >
+                {hasNotification ? (
+                  <Badge
+                    color="primary"
+                    variant="dot"
+                    classes={{ badge: classes.badge }}
+                  >
+                    {nameElement}
```

Comment:
Just inline `nameElement` - it's only one line, you're only using it twice, and it makes the code easier to read since you don't have to go back up to see what it actually is.